Dialyzeabit

zsoci opened the pull request:
Part of neo4j.erl was fixed.
Elvis still complains on too long lines. This seems to be new.
The cleanup shall be continued.

gadgetci commented on src/neo4j.erl:
```diff
@@ -321,14 +326,14 @@ transaction_execute_commit(Neo, Query) ->
   cypher(Neo, Query, null).
 
 %%
 %% http://docs.neo4j.org/chunked/stable/rest-api-cypher.html#rest-api-send-queries-with-parameters
```
According to **Elvis**:
> Line 334 is too long: %% http://docs.neo4j.org/chunked/stable/rest-api-cypher.html#rest-api-send-queries-with-parameters.

gadgetci commented on src/neo4j.erl:
```diff
@@ -1141,7 +1170,7 @@ find_relationship_query(Neo, Index, Query, Ordering) ->
 %%      http://docs.neo4j.org/chunked/milestone/rest-api-unique-indexes.html#rest-api-create-a-unique-node-or-return-fail-fail
```
According to **Elvis**:
> Line 1170 is too long: %%      http://docs.neo4j.org/chunked/milestone/rest-api-unique-indexes.html#rest-api-create-a-unique-node-or-return-fail-fail.

gadgetci commented on src/neo4j.erl:
```diff
@@ -656,7 +663,7 @@ drop_index(Neo, Label, Key) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-indexes.html#rest-api-list-indexes-for-a-label
```
According to **Elvis**:
> Line 664 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-indexes.html#rest-api-list-indexes-for-a-label.

gadgetci commented on src/neo4j.erl:
```diff
@@ -639,7 +646,7 @@ delete_relationship_property(_, _) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-indexes.html#rest-api-create-index
```
According to **Elvis**:
> Line 647 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-indexes.html#rest-api-create-index.

gadgetci commented on src/neo4j.erl:
```diff
@@ -395,7 +402,7 @@ get_node_properties(Node) ->
 %% http://docs.neo4j.org/chunked/stable/rest-api-node-properties.html#rest-api-update-node-properties
```
According to **Elvis**:
> Line 402 is too long: %% http://docs.neo4j.org/chunked/stable/rest-api-node-properties.html#rest-api-update-node-properties.

gadgetci commented on src/neo4j.erl:
```diff
@@ -271,7 +273,9 @@ transaction_commit(T) ->
 %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-execute-statements-in-an-open-transaction-in-rest-format-for-the-return
```
According to **Elvis**:
> Line 273 is too long: %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-execute-statements-in-an-open-transaction-in-rest-format-for-the-return.

gadgetci commented on src/neo4j.erl:
```diff
@@ -583,7 +590,7 @@ get_relationship_properties(Relationship) ->
 %% http://docs.neo4j.org/chunked/stable/rest-api-relationships.html#rest-api-set-all-properties-on-a-relationship
```
According to **Elvis**:
> Line 590 is too long: %% http://docs.neo4j.org/chunked/stable/rest-api-relationships.html#rest-api-set-all-properties-on-a-relationship.

gadgetci commented on src/neo4j.erl:
```diff
@@ -280,7 +284,8 @@ transaction_commit(T, Query) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-rollback-an-open-transaction
```
According to **Elvis**:
> Line 285 is too long: %% @doc http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-rollback-an-open-transaction.

gadgetci commented on src/neo4j.erl:
```diff
@@ -648,7 +655,7 @@ create_index(Neo, Label, PropKeys) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-indexes.html#rest-api-drop-index
```
According to **Elvis**:
> Line 656 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-indexes.html#rest-api-drop-index.

gadgetci commented on src/neo4j.erl:
```diff
@@ -675,7 +682,7 @@ create_constraint(Neo, Label, PropKeys) ->
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-get-a-specific-uniqueness-constraint
 %%
--spec get_constraint(neo4j_root(), binary(), binary()) -> propliststs:proplist() | {error, term()}.
+-spec get_constraint(neo4j_root(), binary(), binary()) -> property_list() | {error, term()}.
```
According to **Elvis**:
> Line 685 is too long: -spec get_constraint(neo4j_root(), binary(), binary()) -> property_list() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -666,7 +673,7 @@ list_indexes(Neo, Label) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-create-uniqueness-constraint
```
According to **Elvis**:
> Line 674 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-create-uniqueness-constraint.

gadgetci commented on src/neo4j.erl:
```diff
@@ -221,7 +223,7 @@ get_properties(Neo) ->
 %%                             )...
 %%
--spec transaction_begin(neo4j_root(), neo4j_transaction_query()) -> proplists:proplist() | {error, term()}.
+-spec transaction_begin(neo4j_root(), neo4j_transaction_query()) -> property_list() | {error, term()}.
```
According to **Elvis**:
> Line 226 is too long: -spec transaction_begin(neo4j_root(), neo4j_transaction_query()) -> property_list() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -261,7 +263,7 @@ transaction_execute(T, Query) ->
 %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-execute-statements-in-an-open-transaction-in-rest-format-for-the-return
```
According to **Elvis**:
> Line 263 is too long: %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-execute-statements-in-an-open-transaction-in-rest-format-for-the-return.

gadgetci commented on src/neo4j.erl:
```diff
@@ -684,7 +691,8 @@ get_constraint(Neo, Label, PropKey) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-get-all-uniqueness-constraints-for-a-label
```
According to **Elvis**:
> Line 692 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-get-all-uniqueness-constraints-for-a-label.

gadgetci commented on src/neo4j.erl:
```diff
@@ -909,7 +937,8 @@ delete_node_index(Neo, Name) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/stable/rest-api-indexes.html#rest-api-list-node-indexes
```
According to **Elvis**:
> Line 938 is too long: %% @doc http://docs.neo4j.org/chunked/stable/rest-api-indexes.html#rest-api-list-node-indexes.

gadgetci commented on src/neo4j.erl:
```diff
@@ -1018,7 +1047,7 @@ create_relationship_index(Neo, Name) ->
 %% @doc
 %%
--spec create_relationship_index(neo4j_root(), binary(), proplists:proplist()) -> neo4j_index() | {error, term()}.
+-spec create_relationship_index(neo4j_root(), binary(), property_list()) -> neo4j_index() | {error, term()}.
```
According to **Elvis**:
> Line 1050 is too long: -spec create_relationship_index(neo4j_root(), binary(), property_list()) -> neo4j_index() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -386,7 +392,8 @@ delete_node(Node) ->
 %%
 %% http://docs.neo4j.org/chunked/stable/rest-api-node-properties.html#rest-api-get-properties-for-node
```
According to **Elvis**:
> Line 393 is too long: %% http://docs.neo4j.org/chunked/stable/rest-api-node-properties.html#rest-api-get-properties-for-node.

gadgetci commented on src/neo4j.erl:
```diff
@@ -300,7 +305,7 @@ transaction_rollback(T) ->
 %%                                      )...
 %%
--spec transaction_execute_commit(neo4j_root(), neo4j_transaction_query()) -> proplists:proplist() | {error, term()}.
+-spec transaction_execute_commit(neo4j_root(), neo4j_transaction_query()) -> property_list() | {error, term()}.
```
According to **Elvis**:
> Line 308 is too long: -spec transaction_execute_commit(neo4j_root(), neo4j_transaction_query()) -> property_list() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -693,7 +701,8 @@ get_uniqueness_constraints(Neo, Label) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-get-all-constraints-for-a-label
```
According to **Elvis**:
> Line 702 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-get-all-constraints-for-a-label.

gadgetci commented on src/neo4j.erl:
```diff
@@ -1220,7 +1249,7 @@ find_relationship_auto_query(Neo, Query) ->
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-configurable-auto-indexes.html#rest-api-create-an-auto-index-for-nodes-with-specific-configuration
 %%
--spec create_node_auto_index(neo4j_root(), proplists:proplist()) -> term() | {error, term()}.
+-spec create_node_auto_index(neo4j_root(), property_list()) -> term() | {error, term()}.
```
According to **Elvis**:
> Line 1252 is too long: -spec create_node_auto_index(neo4j_root(), property_list()) -> term() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -261,7 +263,7 @@ transaction_execute(T, Query) ->
 %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-return-results-in-graph-format
 %%
--spec transaction_commit(neo4j_transaction()) -> proplists:proplist() | {error, term()}.
+-spec transaction_commit(neo4j_transaction()) -> property_list() | {error, term()}.
```
According to **Elvis**:
> Line 266 is too long: -spec transaction_commit(neo4j_transaction()) -> property_list() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -881,7 +906,8 @@ batch(Neo, Request) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/stable/rest-api-indexes.html#rest-api-create-node-index
```
According to **Elvis**:
> Line 907 is too long: %% @doc http://docs.neo4j.org/chunked/stable/rest-api-indexes.html#rest-api-create-node-index.

gadgetci commented on src/neo4j.erl:
```diff
@@ -666,7 +673,7 @@ list_indexes(Neo, Label) ->
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-create-uniqueness-constraint
 %%
--spec create_constraint(neo4j_root(), binary(), [binary()]) -> propliststs:proplist() | {error, term()}.
+-spec create_constraint(neo4j_root(), binary(), [binary()]) -> property_list() | {error, term()}.
```
According to **Elvis**:
> Line 676 is too long: -spec create_constraint(neo4j_root(), binary(), [binary()]) -> property_list() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -181,7 +183,7 @@ get_nodes_by_label(Neo, Label) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/stable/rest-api-node-labels.html#rest-api-get-nodes-by-label-and-property
```
According to **Elvis**:
> Line 184 is too long: %% @doc http://docs.neo4j.org/chunked/stable/rest-api-node-labels.html#rest-api-get-nodes-by-label-and-property.

gadgetci commented on src/neo4j.erl:
```diff
@@ -702,7 +711,7 @@ get_label_constraints(Neo, Label) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-get-all-constraints
```
According to **Elvis**:
> Line 712 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-get-all-constraints.

gadgetci commented on src/neo4j.erl:
```diff
@@ -515,8 +522,8 @@ get_node_labels(Node) ->
 %%
--spec get_relationship(neo4j_root(), neo4j_id() | neo4j_relationship()) -> neo4j_relationship() | {error, term()}.
-get_relationship(_Neo, [{_,_}|_] = Relationship) ->
+-spec get_relationship(neo4j_root(), neo4j_id() | neo4j_relationship() | binary()) -> neo4j_relationship() | {error, term()}.
```
According to **Elvis**:
> Line 525 is too long: -spec get_relationship(neo4j_root(), neo4j_id() | neo4j_relationship() | binary()) -> neo4j_relationship() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -347,7 +352,8 @@ create_node(Neo) ->
 %%
 %% http://docs.neo4j.org/chunked/stable/rest-api-nodes.html#rest-api-create-node-with-properties
```
According to **Elvis**:
> Line 353 is too long: %% http://docs.neo4j.org/chunked/stable/rest-api-nodes.html#rest-api-create-node-with-properties.

gadgetci commented on src/neo4j.erl:
```diff
@@ -639,7 +646,7 @@ delete_relationship_property(_, _) ->
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-indexes.html#rest-api-create-index
 %%
--spec create_index(neo4j_root(), binary(), [binary()]) -> propliststs:proplist() | {error, term()}.
+-spec create_index(neo4j_root(), binary(), [binary()]) -> property_list() | {error, term()}.
```
According to **Elvis**:
> Line 649 is too long: -spec create_index(neo4j_root(), binary(), [binary()]) -> property_list() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -675,7 +682,7 @@ create_constraint(Neo, Label, PropKeys) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-get-a-specific-uniqueness-constraint
```
According to **Elvis**:
> Line 683 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-get-a-specific-uniqueness-constraint.

gadgetci commented on src/neo4j.erl:
```diff
@@ -131,27 +131,28 @@
+-type neo4j_transaction() :: property_list().
 -type neo4j_transaction_query() :: [ Query::binary()
                                    | {Query::binary(), Parameters::binary()}
                                    | {Query :: binary(), Parameters :: binary(), DataFormats::[binary()]}
```
According to **Elvis**:
> Line 144 is too long:                                    | {Query :: binary(), Parameters :: binary(), DataFormats::[binary()]}.

gadgetci commented on src/neo4j.erl:
```diff
@@ -181,7 +183,7 @@ get_nodes_by_label(Neo, Label) ->
 %% @doc http://docs.neo4j.org/chunked/stable/rest-api-node-labels.html#rest-api-get-nodes-by-label-and-property
 %%
--spec get_nodes_by_label(neo4j_root(), binary(), proplists:proplist()) -> [neo4j_node()] | {error, term()}.
+-spec get_nodes_by_label(neo4j_root(), binary(), property_list()) -> [neo4j_node()] | {error, term()}.
```
According to **Elvis**:
> Line 186 is too long: -spec get_nodes_by_label(neo4j_root(), binary(), property_list()) -> [neo4j_node()] | {error, term()}..

gadgetci replied:
According to **Elvis**:
> Line 186 is too long: -spec get_nodes_by_label(neo4j_root(), binary(), property_list()) -> [neo4j_node()] | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -574,7 +581,7 @@ delete_relationship(Relationship) ->
 %% http://docs.neo4j.org/chunked/stable/rest-api-relationships.html#rest-api-get-all-properties-on-a-relationship
 %%
--spec get_relationship_properties(neo4j_relationship() | neo4j_id()) -> proplists:proplist() | {error, term()}.
+-spec get_relationship_properties(neo4j_relationship() | neo4j_id()) -> property_list() | {error, term()}.
```
According to **Elvis**:
> Line 584 is too long: -spec get_relationship_properties(neo4j_relationship() | neo4j_id()) -> property_list() | {error, term()}..

gadgetci replied:
According to **Elvis**:
> Line 584 is too long: -spec get_relationship_properties(neo4j_relationship() | neo4j_id()) -> property_list() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -890,7 +916,9 @@ create_node_index(Neo, Name) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/stable/rest-api-indexes.html#rest-api-create-node-index
```
According to **Elvis**:
> Line 917 is too long: %% @doc http://docs.neo4j.org/chunked/stable/rest-api-indexes.html#rest-api-create-node-index.

gadgetci replied:
According to **Elvis**:
> Line 917 is too long: %% @doc http://docs.neo4j.org/chunked/stable/rest-api-indexes.html#rest-api-create-node-index.

gadgetci commented on src/neo4j.erl:
```diff
@@ -173,7 +174,8 @@ get_relationship_types(Neo) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/stable/rest-api-node-labels.html#rest-api-get-all-nodes-with-a-label
```
According to **Elvis**:
> Line 175 is too long: %% @doc http://docs.neo4j.org/chunked/stable/rest-api-node-labels.html#rest-api-get-all-nodes-with-a-label.

gadgetci replied:
According to **Elvis**:
> Line 175 is too long: %% @doc http://docs.neo4j.org/chunked/stable/rest-api-node-labels.html#rest-api-get-all-nodes-with-a-label.

gadgetci commented on src/neo4j.erl:
```diff
@@ -249,7 +251,7 @@ transaction_begin(Neo, Query) ->
 %%      neo4j:transaction_execute(T1, <<"">>).
 %%
--spec transaction_execute(neo4j_transaction(), neo4j_transaction_query()) -> proplists:proplist() | {error, term()}.
+-spec transaction_execute(neo4j_transaction(), neo4j_transaction_query()) -> property_list() | {error, term()}.
```
According to **Elvis**:
> Line 254 is too long: -spec transaction_execute(neo4j_transaction(), neo4j_transaction_query()) -> property_list() | {error, term()}..

gadgetci replied:
According to **Elvis**:
> Line 254 is too long: -spec transaction_execute(neo4j_transaction(), neo4j_transaction_query()) -> property_list() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -321,14 +326,14 @@ transaction_execute_commit(Neo, Query) ->
 %% http://docs.neo4j.org/chunked/stable/rest-api-cypher.html#rest-api-send-queries-with-parameters
 %%
--spec cypher(neo4j_root(), binary(), proplists:proplist()) -> cypher_result() | {error, term()}.
+-spec cypher(neo4j_root(), binary(), property_list() | null) -> {cypher_result()} | {error, term()}.
```
According to **Elvis**:
> Line 336 is too long: -spec cypher(neo4j_root(), binary(), property_list() | null) -> {cypher_result()} | {error, term()}..

gadgetci replied:
According to **Elvis**:
> Line 336 is too long: -spec cypher(neo4j_root(), binary(), property_list() | null) -> {cypher_result()} | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -515,8 +522,8 @@ get_node_labels(Node) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/stable/rest-api-relationships.html#rest-api-get-typed-relationships
```
According to **Elvis**:
> Line 523 is too long: %% @doc http://docs.neo4j.org/chunked/stable/rest-api-relationships.html#rest-api-get-typed-relationships.

gadgetci replied:
According to **Elvis**:
> Line 523 is too long: %% @doc http://docs.neo4j.org/chunked/stable/rest-api-relationships.html#rest-api-get-typed-relationships.

gadgetci commented on src/neo4j.erl:
```diff
@@ -1228,7 +1257,9 @@ create_node_auto_index(Neo, Config) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-configurable-auto-indexes.html#rest-api-create-an-auto-index-for-relationships-with-specific-configuration
```
According to **Elvis**:
> Line 1258 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-configurable-auto-indexes.html#rest-api-create-an-auto-index-for-relationships-with-specific-configuration.

gadgetci replied:
According to **Elvis**:
> Line 1258 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-configurable-auto-indexes.html#rest-api-create-an-auto-index-for-relationships-with-specific-configuration.

gadgetci commented on src/neo4j.erl:
```diff
@@ -271,7 +273,9 @@ transaction_commit(T) ->
 %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-execute-statements-in-an-open-transaction-in-rest-format-for-the-return
 %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-return-results-in-graph-format
```
According to **Elvis**:
> Line 274 is too long: %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-return-results-in-graph-format.

gadgetci replied:
According to **Elvis**:
> Line 274 is too long: %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-return-results-in-graph-format.

gadgetci commented on src/neo4j.erl:
```diff
@@ -1220,7 +1249,7 @@ find_relationship_auto_query(Neo, Query) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-configurable-auto-indexes.html#rest-api-create-an-auto-index-for-nodes-with-specific-configuration
```
According to **Elvis**:
> Line 1250 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-configurable-auto-indexes.html#rest-api-create-an-auto-index-for-nodes-with-specific-configuration.

gadgetci replied:
According to **Elvis**:
> Line 1250 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-configurable-auto-indexes.html#rest-api-create-an-auto-index-for-nodes-with-specific-configuration.

gadgetci commented on src/neo4j.erl:
```diff
@@ -648,7 +655,7 @@ create_index(Neo, Label, PropKeys) ->
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-indexes.html#rest-api-drop-index
 %%
--spec drop_index(neo4j_root(), binary(), binary()) -> propliststs:proplist() | {error, term()}.
+-spec drop_index(neo4j_root(), binary(), binary()) -> property_list() | {error, term()}.
```
According to **Elvis**:
> Line 658 is too long: -spec drop_index(neo4j_root(), binary(), binary()) -> property_list() | {error, term()}..

gadgetci replied:
According to **Elvis**:
> Line 658 is too long: -spec drop_index(neo4j_root(), binary(), binary()) -> property_list() | {error, term()}..

gadgetci commented on src/neo4j.erl:
```diff
@@ -261,7 +263,7 @@ transaction_execute(T, Query) ->
 %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-execute-statements-in-an-open-transaction-in-rest-format-for-the-return
 %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-return-results-in-graph-format
```
According to **Elvis**:
> Line 264 is too long: %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-return-results-in-graph-format.

gadgetci replied:
According to **Elvis**:
> Line 264 is too long: %%      http://docs.neo4j.org/chunked/stable/rest-api-transactional.html#rest-api-return-results-in-graph-format.

gadgetci commented on src/neo4j.erl:
```diff
@@ -711,7 +720,8 @@ get_constraints(Neo) ->
 %%
 %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-drop-constraint
```
According to **Elvis**:
> Line 721 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-drop-constraint.

gadgetci replied:
According to **Elvis**:
> Line 721 is too long: %% @doc http://docs.neo4j.org/chunked/milestone/rest-api-schema-constraints.html#rest-api-drop-constraint.

gadgetci commented on src/neo4j.erl:
```diff
@@ -574,7 +581,7 @@ delete_relationship(Relationship) ->
 %%
 %% http://docs.neo4j.org/chunked/stable/rest-api-relationships.html#rest-api-get-all-properties-on-a-relationship
```
According to **Elvis**:
> Line 582 is too long: %% http://docs.neo4j.org/chunked/stable/rest-api-relationships.html#rest-api-get-all-properties-on-a-relationship.

gadgetci replied:
According to **Elvis**:
> Line 582 is too long: %% http://docs.neo4j.org/chunked/stable/rest-api-relationships.html#rest-api-get-all-properties-on-a-relationship.

dmitriid commented:
OMG. Thank you! I'll look at it later in the day (time permitting)

zsoci commented:
Don't worry, we are not on hurry. I have fixed only the cypher call thread as that was needed to have a clean dialyzer run for me. It seems, elvis is making a lot of suggestions. As it hits all the areas of the code, I have not fixed them. There is a main change that is proplists:proplist(). I changed it everywhere to list(proplists:property()) as the original does not seem to be supported. Just take your time. And thanks very much for this contribution to neo4j!

dmitriid commented:
Merged into master.
